serving/src/main/java/feast/serving/specs/CachedSpecService.java

```diff
@@ -103,69 +103,99 @@ public FeatureSetSpec getFeatureSetSpec(String featureSetRef) throws ExecutionEx
     return featureSetCache.get(featureSetRef);
   }
 
+  /**
+   * Get FeatureSetSpecs for the given features references. See {@link #getFeatureSets(List,
+   * String)} for full documentation.
+   */
+  public List<FeatureSetRequest> getFeatureSets(List<FeatureReference> featureReferences) {
+    return getFeatureSets(featureReferences, "");
```

Comment:
Out of curiosity, is there a reason that the default project could not be used here?

Reply:
When `project` is `""` we want to revert the existing behavior of inspecting the Feature References for the project use in the request, in addition to the default project.

Updated docs to clarify this.